ci/pylint.yml

```
@@ -17,6 +17,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pylint
-    - name: Test with pytest
+        pip install pylint-exit
+    - name: Run Pylint
       run: |
-        pylint `ls -R|grep .py$|xargs`
+        pylint $(find . -type f -name "*.py" -not -path './build*' | xargs) || pylint-exit $?
```

A simpler solution is to use to call `pylint` directly from find:

```suggestion
        find -regextype egrep -regex '(.*.py)$' -exec pylint {} \;
```

I use egrep type with that format because it can be extended, for example, to

```sh
find -regextype egrep -regex '(.*.py|.*.py.in)$' -exec pylint {} \;
```

I do not see the purpose of excluding the `build` directory. In which circumstances would you add the `build` to the repository?